.github/workflows/percy-demo-tests.yml

```
@@ -65,4 +75,21 @@ jobs:
           CYPRESS_demoTests: true 
         with:
           group: "UI - Chrome"
-          command: npx percy exec -- npx cypress run --record
\ No newline at end of file
+          command: npx percy exec -- npx cypress run --record
+
+      - name: Cypress run test functional
+        if: github.event.inputs.testsKind != "percy"
```

Why don't check test kind directly?
```suggestion
        if: contains(fromJSON('["functional", "all"]'), github.event.inputs.testsKind)
```